.gitignore

```diff
@@ -127,6 +127,3 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
-
-# Plugin configuration file
-configuration.json
```

Comment:
why delete this?

Reply:
For this refactor I added the configuration.json file that is also a part of the mg-scripts repo into this repo. This necessitated removing this line from .gitignore so it could be added. This information is needed for running tests. In the past we were able to avoid adding configuration.json by hardcoding the data-structure in the tests. I wanted to avoid this in the new tests.

Reply:
Does this mean that now we have 2 config files one provided by mg-scripts and another by this repo?

Reply:
Yes. The alternative is to keep the data-structure as a const in the test code.